app.js

```diff
@@ -0,0 +1,138 @@
+    imgSrc: './image/SnapshootPortfoliop3.png',
+    linkstolv: 'See Lives ',
+    linkstos: 'See Sources ',
+  },
+  {
+    id: 1,
+    name: 'Uber Navigation',
+    techolgies: ['HTML', 'CSS', 'JAVAScript', 'Ruby', 'BootStrap'],
+    Dtecholgies: ['Uber', 'Lead Dev', '2015'],
+    description:
+     [" Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has  been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it 1960s. Lorem Ipsum is simply dummy text of the printing and  typesetting industry. ",
+       " Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it 1960s with the releorem Ipsum is simply dummy text  of the printing and typesetting industry. Lorem Ipsum han printer took a galley of type and scrambled it 1960 "],
+    imgSrc: './image/SnapshootPortfoliop4.png',
+    linkstolv: 'See Lives ',
+    linkstos: 'See Sources ',
+  },
+];
+
+// SELECT ELEMENTS
+const cardsE = document.querySelector('.popup2');
+
+// RENDER PRODUCTS
+function renderProdcuts(num) {
+  const carde = Object.values(cards)[num];
+  cardsE.innerHTML = `
+  <div class="header">
+  <div class="left">
+      <h3>${carde.name}</h3>
+      <ul>
+          <li>${carde.Dtecholgies[0]}</li>
+          <li>${carde.Dtecholgies[1]}</li>
+          <li>${carde.Dtecholgies[2]}</li>
+      </ul>
+  </div>
+  <div class="right">
+  <a href="#" onclick="toggle()"><img src="./image/Icon2.png" alt=""></a>
+      
+  </div>
+  </div>
+  <div class="middle">
+  
+      <img src="${carde.imgSrc}" alt="">
+  </div>
+
+  <div class="contents">
+      <div class="left">
+          <p>
+              ${carde.description[0]}
+          </p>
+          <div class="top">
+              <p >
+              ${carde.description[1]}
+              </p>
+           </div>
+          
+                  
+      </div>
+      <div class="right">
+          <div class="links">
+              <ul>
+                  <li>${carde.techolgies[0]}</li>
+                  <li>${carde.techolgies[1]}</li>
+                  <li>${carde.techolgies[2]}</li>
+                  <li>${carde.techolgies[3]}</li>
+                  <li>${carde.techolgies[4]}</li>
+              </ul>
+          </div>
+          <div class="lines">
+            <br>
+             <hr>
+          </div>
+          <div>
+              <img src="Vector 4.png" alt="">
+          </div>
+          <div class="buttons">
+          <button class="links-live">${carde.linkstolv}<img src="./image/See-live-Icon.png" alt=""> </button>
+          <button class="links-source">${carde.linkstos}<img src="./image/see-sourec-Vector.png" alt=""></button>
+          </div>
+      </div>
+      
+  </div>
```

Comment:
Great idea to generate the cards dynamically with the use of identifiable digits. 🚀.  There are only a few changes I would want to suggest ♻️. 

- [ ] Kindly fix the `width` and `height` of your Card popup on `Mobile` as it s smaller than it appears in the given template design. Following templates strictly is always important as it will help satisfy clients and designers in your future job. 🧑‍💼 
- [ ] Please fix the scroll of the main page when popup is active as it does not give a good user experience and can cause distraction at the background for users. Please check below ⬇️ 
![about](https://github.com/ethred/Portfolio/assets/117144028/f2737379-c2e8-4ddd-9c5d-19c98df8f63d)

### Figma design ⬇️ 
![image](https://github.com/ethred/Portfolio/assets/117144028/154600f2-af9e-40bb-8b4b-96f7855d412d)

### Your design ⬇️
![image](https://github.com/ethred/Portfolio/assets/117144028/466f9ef5-89d3-4685-a989-ababc0bc3cae)

Reply:
- [ ] As the previous reviewer requested, kindly position the popup correctly following the Figma design. It will make your webpage more beautiful 🌷